inspire/dojson/hep/fields/bd76x78x.py

```diff
@@ -28,31 +28,42 @@
 
 
 @hep.over('publication_info', '^773..')
-@utils.for_each_value
-@utils.filter_values
 def publication_info(self, key, value):
     """Publication info about record."""
-    year = ''
-    recid = ''
-    if 'y' in value:
-        year = int(value.get('y'))
-    if '0' in value:
-        recid = int(value.get('0'))
-    return {
-        'recid': recid,
-        'page_artid': value.get('c'),
-        'journal_issue': value.get('n'),
-        'conf_acronym': value.get('o'),
-        'journal_title': value.get('p'),
-        'reportnumber': value.get('r'),
-        'confpaper_info': value.get('t'),
-        'journal_volume': value.get('v'),
-        'cnum': value.get('w'),
-        'pubinfo_freetext': value.get('x'),
-        'year': year,
-        'isbn': value.get('z'),
-        'note': value.get('m'),
-    }
+
+    def get_value(value):
```

Comment:
I am checking again the previous code and it is working good (!). It gives back the correct results. I am going to remove the file from the commit as long as there is no problem. I don't get why I had this problem before :/